### ISSUE TITLE
User authentication

### ISSUE THREAD

**Issue by murtoM** [murtoM/invekotin#1]
Add initial support for user authentication (for start it could be for a hard coded user account) using Passportjs.

There probably wont be any kind of special landing page for clients who have not authenticated, so the Router could redirect everything to the login page unless the user is authenticated (create another issue once initial auth support is done).

**Comment by murtoM** [comment]
Remove the usage of `passport-local-mongoose` and instead use bare `passport-local` with bespoke thingiemajigs.

**Comment by murtoM** [comment]
It should be fine for now.
